$NetBSD$

NetBSD uses mandoc since netbsd-6.

Also, remove mancat since it unnecessarily expands @mandir@ into
@mandir@/man and we already provide a complete --mandir.

--- configure.orig	2014-08-06 14:43:42.000000000 +0000
+++ configure
@@ -2000,7 +2000,7 @@ if test "$mandir" = '${prefix}/man'; the
 	mandir=/usr/local/man
     fi
 fi
-mandir="$mandir/$mancat"
+mandir="$mandir"
 USE_GROFF='USE_GROFF=no'
 if expr "$MAN" : 'dcc\.0' >/dev/null; then
    if groff -Tascii -mtty-char -mdoc </dev/null >/dev/null; then
